OpenRA.Game/Map/Map.cs

```diff
@@ -265,7 +266,7 @@ public static string ComputeUID(IReadOnlyPackage package)
 			try
 			{
 				foreach (var filename in contents)
-					if (filename.EndsWith(".yaml") || filename.EndsWith(".bin") || filename.EndsWith(".lua"))
+					if (filename.EndsWith(".yaml") || filename.EndsWith(".bin") || filename.EndsWith(".lua") || (format >= 12 && filename == "map.yaml"))
```

Comment:
Why does `filename.EndsWith(".yaml")` not cover `map.yaml` already?

Reply:
I'm pretty sure that this should be checking `map.png` instead.

Reply:
fixed